README.md

```diff
@@ -79,6 +79,247 @@ For `testStart` and `testEnd`, the corresponding test object is passed to the re
 
 When `runStart`, `suiteStart` and `testStart` are emitted, the `status`, `runtime` and `errors` attributes are `undefined`.
 
+## Example
+
+Lets take the below testsuite and see how the reporting shoud be, based on the standard presented above. For implementations examples please check [Usage](#usage) and [Integrations](#integrations) below for implementations in real tools.
+
+```js
+describe('suite 1', function() {
+  describe('suite 2', function() {
+    it('test2', function() {
+      expect(true).to.be.true;
+    });
+  });
+
+  it('test1', function() {
+    expect(true).to.be.true;
+    expect(true).to.be.false;
+  });
+});
+```
+
+The emitted events and data will be in the following order:
+
+- **runStart**: 
```

Comment:
Having an example at this level of detail makes sense, but its too long for this document. Let's extract that to a separate file (`example.md`? `docs/example.md`?). Maybe we can include a compressed version here, something like...

```md
* **runStart**: `{ name: undefined, childSuites: [Suite('suite1')] }`
* **suiteStart**: `{ name: 'suite1', childSuites: [Suite('suite2')] }`
...
```